packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -8,12 +8,22 @@ import { compareSDKVersions } from './utils';
  * @internal
  * Limited to Microsoft-internal use
  */
-export function ensureInitialized(...expectedFrameContexts: string[]): void {
+export function ensureInitializeCalled(): void {
   if (!GlobalVars.initializeCalled) {
     throw new Error('The library has not yet been initialized');
   }
+}
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function ensureInitialized(...expectedFrameContexts: string[]): void {
+  if (!GlobalVars.initializeCompleted || !GlobalVars.frameContext) {
```

Comment:
Why was it necessary to also check that frameContext was set? is there a case where that might be an issue when initializeCompleted is true, or just trying to be careful?

Reply:
I removed it. It was mostly being careful.